src/index.ts

```diff
@@ -1 +1 @@
-export {};
+export * from './parser';
```

Comment:
At the moment it's fine, but in the future please return only functions from `diff` file :) I think that file will be called with main functionality.